### PR TITLE
Added 2.3/ from forked project. Renamed 2.3.2 to 2.3.1 and added plug…

### PR DESCRIPTION
**Pull request by jhocc**
…in installing. @pemacc

### CODE REVIEW

File: 2.3/2.3.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
+FROM java:8-jre
+
```
Comment (pemacc) [inline 2.3/2.3.1/Dockerfile]: can we get elasticsaerch 2.3.1 image here

File: 2.3/2.3.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ENV ELASTICSEARCH_VERSION 2.3.1
+ENV ELASTICSEARCH_REPO_BASE http://packages.elasticsearch.org/elasticsearch/2.x/debian
+
+RUN echo "deb $ELASTICSEARCH_REPO_BASE stable main" > /etc/apt/sources.list.d/elasticsearch.list
+
+RUN set -x \
+	&& apt-get update \
+	&& apt-get install -y --no-install-recommends elasticsearch=$ELASTICSEARCH_VERSION \
+	&& rm -rf /var/lib/apt/lists/*
+
```
Comment (pemacc) [inline 2.3/2.3.1/Dockerfile]: Is there existing elasticsearch 2.3.1 image? If so can we build on top of it?

Comment (jhocc) [inline 2.3/2.3.1/Dockerfile]: Nope. No existing 2.3.1 image exists. We had to lock the version @ 2.3.1, which is the version that jimmy is using.

File: 2.3/2.3.1/Dockerfile (hunk 1 above)
Comment (jhocc) [inline 2.3/2.3.1/Dockerfile]: none exists, there is a 2.3.2 version, but we needed 2.3.1 which is the version jimmy is using.

**Comment by jhocc** [comment]
@pemacc updated.

**Pull request merged**